### PR TITLE
feat: expo upgrade to 49

### PR DESCRIPTION
**Pull request by sakul-budhathoki**
- Expo update to 49
- Webpack has been updated to v5 due to `@expo/webpack-config": "^19.0.0` upgrade

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const MomentLocalesPlugin = require("moment-locales-webpack-plugin");
 const { WebpackDeduplicationPlugin } = require("webpack-deduplication-plugin");
 module.exports = async function (env, argv) {
   const config = await createExpoWebpackConfigAsync(env, argv);
+  config.externals = ["crypto"];
```
Comment (sakul-budhathoki) [inline webpack.config.js:8]: These changes resolve the following error; Let me know if this is the correct way to fix this error. Thanks.

```
ERROR in ./node_modules/@cosmjs/crypto/build/pbkdf2.js:14:19
Module not found: Can't resolve 'crypto'

ERROR in ./node_modules/@cosmjs/ledger-amino/node_modules/@cosmjs/crypto/build/pbkdf2.js:14:23
Module not found: Can't resolve 'crypto'
```

File: .env [hunk 2]
```diff
@@ -30,4 +30,6 @@ PRICES_SERVICE_URI=prices.mainnet.teritori.com:443
 
 IMG_PROXY_URL=https://imgproxy.tools.teritori.com/insecure/
 
+GENERATE_SOURCEMAP=false
```
Comment (sakul-budhathoki) [inline .env:33]: This env suppresses the below warnings:

```
WARNING in ./node_modules/osmojs/node_modules/@cosmjs/proto-signing/build/pubkey.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/pubkey.ts' file: Error: ENOENT: no such file or directory, open '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/pubkey.ts'

WARNING in ./node_modules/osmojs/node_modules/@cosmjs/proto-signing/build/registry.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/registry.ts' file: Error: ENOENT: no such file or directory, open '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/registry.ts'
```

**Comment by n0izn0iz** [comment]
note: we can't switch the web bundler to metro as recommended since metro does not support dynamic imports

**Comment by n0izn0iz** [comment]
I fixed the env warning

File: webpack.config.js (hunk 1 above)
Comment (n0izn0iz) [reply webpack.config.js:8]: it might be better to add a fallback, where did you find this solution?

**Comment by omniwired** [comment]
I found an issue with the reordering of the apps in the dAppStore.

File: .env (hunk 2 above)
Comment (n0izn0iz) [reply .env]: I changed the webapck config to ignore warnings (like in previous expo versions) instead of not generating sourcemaps

**Pull request merged**